clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -461,13 +461,79 @@ check_function_markers()
     return true;
 }
 
+bool
+check_repeated_syscall_with_same_pc()
+{
+    constexpr addr_t ADDR_ONE = 0x7fcf3b9dd9e9;
```

Comment:
The mock addresses can be something simple instead of this hex value.

Reply:
Reminder for this unresolved comment.

Reply:
Just kept this one and removed the other ones based on comment below.